lib/src/chip/chip.component.ts

```diff
@@ -0,0 +1,38 @@
+import { Component, Input, ViewEncapsulation, OnInit } from '@angular/core';
+import { anyToBoolean } from '../util';
+
+@Component({
+    selector: 'hc-chip',
+    template: `<div [ngClass]="chipType"><ng-content></ng-content></div>`,
+    styleUrls: ['./chip.component.scss'],
+    encapsulation: ViewEncapsulation.None
+})
+
+export class ChipComponent implements OnInit {
+
+    _action: boolean = false;
+    @Input() color: string = 'neutral';
+    chipType: string = 'chip';
+
+    constructor() {}
+
+    ngOnInit() {
+        this.chipType += ' ' + this.color;
+    }
+
+    @Input()
+    get action(): boolean {
```

Comment:
- If the color property were changed, that change wouldn't be reflected unless you also update the action property. 
- I think it would be possible to toggle the 'close' class based on action without having to use getters/setters